config/env/production/middlewares.js

```diff
@@ -0,0 +1,22 @@
+module.exports = [
+  "strapi::errors",
+  "strapi::security",
```

Comment:
https://github.com/strapi/strapi/tree/v4.0.4/packages/providers/upload-cloudinary#security-middleware-configuration

Config seems to be slightly different in the provider example.

Reply:
You are right. I have missed reading that documentation and came to this config myself.

Reply:
@insuusvenerati Fixed now